addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18153,3 +18153,23 @@ msgstr ""
 msgctxt "#38025"
 msgid "Choose information provider"
 msgstr ""
+
+#: system/settings/rbp.xml
+msgctxt "#38026"
+msgid "Decode the stereo stream from 3D files"
+msgstr ""
+
+#: system/settings/rbp.xml
+msgctxt "#38027"
+msgid "This option decodes frames for both eyes from video containing Multiview Video Coding (MVC) as typically found on 3D Blu-ray. Disabling may improve performance if you don't require 3D"
```

Comment:
does enabling this option have a performance impact on every kind of playback, or only on certain file types? Oh, and a dot is missing at the end of the second sentence :)
A contextual note would also be welcome, maybe something like `#. Description of setting "Decode the stereo stream from 3D files" with label #38026`. Same with other labels. There is a short guide on top of the strings.po file on how to handle new labels (just as a FYI).